functions/man.fish

```diff
@@ -1,3 +1,14 @@
 function man --wraps man -d "Run man with added colors"
-    cless /usr/bin/man $argv
+    set --local --export MANPATH $MANPATH
```

Comment:
I don't understand what the point of adding fish_manpath to MANPATH is.

Reply:
> Make Fish's manuals available in man, I assume that is pretty obvious...

Oh my gosh, I just realized I never knew about this because this whole time I've messed up MANPATH this my own plugun! Thank you for showing me this!!

Reply:
FYI there is another colored man plugin for fish: https://github.com/decors/fish-colored-man

Reply:
Nice, they've included this functionality all along.